### DISCUSSION TITLE
Help me vercel 404 not loading

### DISCUSSION THREAD

**Discussion by pretzer25** [vercel/community#5127] [Help] [answered]
### Summary

when i make a deployment it keeps saying 404, even though im 100% sure the link is rigth, i clicked it from vercel's site!

### Example

https://myharhar.vercel.app/

### Steps to Reproduce

make a new deployment, doesnt always work thou

**Comment by ghost** [comment]
Hey there! What Framework Preset are you using when you deploy your site?

**Comment by ghost** [comment, marked as answer]
If you Framework Preset is set to "other", please make sure you have a valid `index.html` file in your directory/repository.
